kitsune/wiki/templates/wiki/edit.html

```diff
@@ -22,6 +22,12 @@
 
     <article id="edit-document" {% if locked %}class="locked"{% endif %} data-slug="{{ document.slug }}">
       <h1>{{ _('<em>Editing</em> {title}')|fe(title=document.title) }}</h1>
+      {% if document_needs_change %}
+        <p>
+          {% trans comment=_(document.needs_change_comment) %} <b>Needs Change:</b> {{comment}} {% endtrans %}
+        </p>
+      {% endif %}
```

Comment:
You should simply use `{% if doc.needs_change %}` instead of introducing a new variable name.